.npmrc

```diff
@@ -11,3 +11,5 @@ public-hoist-pattern[]=*nuxt*
 public-hoist-pattern[]=defu
 public-hoist-pattern[]=ufo
 public-hoist-pattern[]=unctx
+
+shamefully-hoist=true
```

Comment:
Preferably not shamefully hoist, put all necessary packages as hoist patterns instead?

Reply:
yes, I'd agree. and I _think_ we should be able to avoid hoisting with latest nuxt...